docs/upgrade.md

```diff
@@ -168,6 +169,13 @@ In Marionette 3, the HTML will be:
 </div>
 ```
 
+### Arguments for Lifecycle Events
+
+The arguments to all lifecycle events were completely changed. All events will
```

Comment:
I'm honestly not too worried about the comparison.. the args weren't super well documented before and for these lesser used events.. probably saw little use.  I just don't want to worry anyone looking to upgrade, when most of the heavily used events didn't change.